rules/when-to-use-anchoring/rule.md

```diff
@@ -0,0 +1,53 @@
+---
+type: rule
+title: Do you know when to use anchoring?
+uri: when-to-use-anchoring
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  guid: 69b80ab3-e4fa-4624-93a5-3ec6c01141d8
+created: 2024-05-03T18:29:33.0000000Z
+related: 
+  - share-the-agenda
+  - stick-to-the-agenda-and-complete-the-meetings-goal
+  - anchoring-sales-strategy
+redirects: []
+  
+
+---
+Anchoring is a cognitive bias where individuals rely heavily on the first piece of information they encounter (the "anchor") when making decisions or judgments. This initial information then influences subsequent thinking and judgments, often leading to skewed conclusions or estimations. Anchoring can occur consciously or unconsciously and can significantly impact decision-making processes across various contexts, from negotiations and purchases to problem-solving and risk assessment.
+
+While anchoring can be a powerful technique in certain situations, it's essential to consider its potential drawbacks and the context in which it is used to achieve the most postitive outcome to a situation. During meetings, senior people should give their opinions last to avoid creating bias to foster better collaboration and discussion.
+
+<!--endintro-->
+
+`youtube: https://youtu.be/JL4OoKJyNrc`
+**Video: Jeff Bezos - Talking about bias in meetings**
```

Comment:
@Tanya-Leahy 

- [ ] 1. Follow https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions here ;)